corehq/apps/sso/models.py

```diff
@@ -0,0 +1,167 @@
+from django.db import models
+
+from corehq.apps.accounting.models import BillingAccount
+from corehq.apps.sso import certificates
+from corehq.apps.sso.exceptions import ServiceProviderCertificateError
+
+
+class IdentityProviderType(object):
```

Comment:
nit: `object` base class is no longer required since Python 3 (it's implied, and all classes are new-style in Python 3)
```suggestion
class IdentityProviderType:
```

This applies to other places throughout this PR.